README.md

```diff
@@ -302,6 +302,37 @@ fetcher
 
 For requests from the server, the config object is simply passed into the service being called.
 
+## Using Context Picker
```

Comment:
We don't talk about context variables in depth anywhere else so this would be a good section for that.